build/moz.configure/bindgen.configure

```diff
@@ -4,10 +4,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-# cbindgen is needed by the style system build and webrender.
-cbindgen_is_needed = depends(build_project)(lambda build_project: build_project != 'js')
```

Comment:
Note: I presume this might need some announcement on the JS mailing list once we merge the rust-frontend project in mozilla-central. As from what I understand, this unconditionally add `cbindgen` as a dependency.